src/App.jsx

```diff
@@ -1,3 +1,79 @@
+import { useEffect, useState } from "react"
+import { Feed } from "./components/Feed"
+import { Form } from "./components/Form"
+import "./App.css"
+
 export const App = () => {
-  return <div>Find me in src/app.jsx!</div>;
-};
+  //Initialize state for storing dafault data
+  const [thoughts, setThoughts] = useState([])
+
+  //Define API endpoints
+  const thoughtsURL = "https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts"
+
+  //Function to fetch data from API
+  const fetchHappyThoughts = () => {
+    fetch(thoughtsURL)
+      .then((result) => result.json())
```

Comment:
maybe here you can throw a new error if response.ok is falsy, just like what you did for POST request.